fxcore/assetlist.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "fxcore",
+  "assets": [
+    {
+      "description": "The native staking token of the Function X",
+      "denom_units": [
+        {
+          "denom": "FX",
+          "exponent": 0
+        }
+      ],
+      "base": "fx",
```

Comment:
the base must refer to some denom_unit (case-sensitive), and so I'm thinking this should probably be capitalized. But if that actual base unit is fx, then all other cases should be lower-cased (i.e., the display, staking token denom, and fee token denom)

Reply:
Thanks. We'll go with this. Just fyi pretty much all other chains have all lower case for the base denom--just thought I'd mention in case we ever see issue with a mismatch.